backend/attendance-service/src/main/java/org/egov/validator/AttendeeServiceValidator.java

```diff
@@ -27,47 +30,70 @@ public class AttendeeServiceValidator {
 
     @Autowired
     private MDMSUtils mdmsUtils;
+
     public void validateAttendeeCreateRequestParameters(AttendeeCreateRequest attendeeCreateRequest) {
-            List<IndividualEntry> attendeeList = attendeeCreateRequest.getAttendees();
+        List<IndividualEntry> attendeeList = attendeeCreateRequest.getAttendees();
 
-            if (attendeeList == null || attendeeList.size() == 0) {
-                throw new CustomException("ATTENDEES", "ATTENDEE object is mandatory");
-            }
+        if (attendeeList == null || attendeeList.size() == 0) {
+            log.error("ATTENDEE Object is empty in attendee request");
+            throw new CustomException("ATTENDEES", "ATTENDEE object is mandatory");
+        }
 
-            String tenantId = attendeeList.get(0).getTenantId();
-            for (IndividualEntry attendee : attendeeList) {
+        String tenantId = attendeeList.get(0).getTenantId();
+        for (IndividualEntry attendee : attendeeList) {
 
-                //validate request parameters for each attendee object
+            //validate request parameters for each attendee object
 
-                if (ObjectUtils.isEmpty(attendee)) {
-                    throw new CustomException("ATTENDEE", "ATTENDEE is mandatory");
-                }
-                if (StringUtils.isBlank(attendee.getRegisterId())) {
-                    throw new CustomException("REGISTER_ID", "Register id is mandatory");
-                }
+            if (ObjectUtils.isEmpty(attendee)) {
+                log.error("ATTENDEE Object is empty in attendee request");
+                throw new CustomException("ATTENDEE", "ATTENDEE is mandatory");
+            }
```

Comment:
Check if this is required

Reply:
Check required in a very niche case as discussed. Hence Removed.